_pages/pedrosozin.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: pedrosozin
+pageUrl: pedrosozin.github.io
+timestamp: 2017-04-13
+---
```

Comment:
Hey @pedrosozin, add a newline to this file to make your builds pass. Nice work!